pkgs/tools/networking/network-manager/default.nix

```diff
@@ -76,6 +76,13 @@ in stdenv.mkDerivation rec {
       url = https://bugzilla.gnome.org/attachment.cgi?id=372953;
       sha256 = "1crjplyiiipkhjjlifrv6hhvxinlcxd6irp9ijbc7jij31g44i0a";
     })
+    (fetchpatch {
+      # https://gitlab.gnome.org/GNOME/network-manager-applet/issues/12
+      # should be fixed in 1.13.0
```

Comment:
1.13.0 would be unstable, plus Lubomir promised 1.12.2.

Reply:
updated the comment.